cmd/ws/main.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/go-kit/kit/log"
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
+	"github.com/mainflux/mainflux"
+	manager "github.com/mainflux/mainflux/manager/client"
+	adapter "github.com/mainflux/mainflux/ws"
+	"github.com/mainflux/mainflux/ws/api"
+	"github.com/mainflux/mainflux/ws/nats"
+	broker "github.com/nats-io/go-nats"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	defPort       = "8180"
+	defNatsURL    = broker.DefaultURL
+	defManagerURL = "http://localhost:8180"
+	envPort       = "MF_WS_ADAPTER_PORT"
+	envNatsURL    = "MF_NATS_URL"
+	envManagerURL = "MF_MANAGER_URL"
+	topic         = "src.ws"
```

Comment:
As far as I can see this constant is not used at all, so either use it or drop it. More general notes, since it is obvious that some naming standard needs to be introduced:

- Rename to `subject` just to avoid ambiguities. Also rename it in the HTTP adapter since that's the "source of ambiguity".
- Should this be the subject, or the prefix? Aren't we naming the subjects as `src.<protocol>.<channel>`?